spyder/plugins/outlineexplorer/editor.py

```diff
@@ -14,12 +21,13 @@ def __init__(self, editor, fname):
         super(OutlineExplorerProxyEditor, self).__init__()
         self._editor = editor
         self.fname = fname
+        self.symbol_info = {}
+        self.current_tree = IntervalTree()
```

Comment:
Are these two attributes really necessary? They don't seem to be used elsewhere in this PR.